funding/manager.go

```diff
@@ -3919,7 +3926,17 @@ func (f *Manager) handleChannelReadyReceived(channel *channeldb.OpenChannel,
 		peerAlias = &foundAlias
 	}
 
-	err := f.addToRouterGraph(channel, scid, peerAlias, nil)
+	// A channel is an announced channel if the ChannelFlag is set and the
+	// number of required confirmations for the funding tx matches the
+	// minimum required depth for a channel of six confirmations. A channel
+	// can be usable before six confirmations but we still keep the channel
+	// unannounced even for public channels until six confirmations are
+	// reached. Public zero-conf channels are also not announced to the
+	// broader network until six confirmations have been reached.
+	shouldAnnounce := channel.ChannelFlags&lnwire.FFAnnounceChannel == 1 &&
+		channel.NumConfsRequired >= 6 && peerAlias == nil
```

Comment:
Agree maybe not that intuitive, but for normal channels we only execute this function if the channel has reached the minimum_accepted depth or am I misunderstanding something ?  
Nonetheless I will base it on mined blocks rather than the `min_accepted_height`

Reply:
Ah I see, this should probably have a more clear comment explicitly saying that 6 confs have passed in this case

Reply:
Cool, I decided to go with the check as it is and added a more detailed comment. Fetching chain data here would overcomplicate things in my view because we would have to fetch chain data here (introducing some level of indirection) wdyt ?